_content/essays/opening-door-of-dharma_chokyi-lodro.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Opening the Door of Dharma"
+authors:
+  - chokyi-lodro
+translator: Adam Pearcey
+external_url: "https://www.lotsawahouse.org/tibetan-masters/jamyang-khyentse-chokyi-lodro/opening-the-door-of-dharma"
+drive_links:
+  - "https://drive.google.com/file/d/16-wEUk_mZnoYNMcMD4b4mcAFeqd7hGD2/view?usp=sharing"
+course: tibetan
+tags:
+  - dialogue
+year: 2020
+publisher: lotsawa-house
+pages: 10
+---
+
+> There are thus a great many systems of Dharma teaching in Tibet,
```

Comment:
This looks like a poem to me. So two spaces `  ` after each line and `subcat: poetry`